### PR TITLE
bind ctrl + s to screenshot by default

### PR DESCRIPTION
**Pull request by joshuapeters**
added ctrl + s as a keybind for the screenshot plugin as requested by issue #2483

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/plugins/screenshot/ScreenshotInput.java [hunk 1]
```diff
@@ -58,6 +66,9 @@ public void keyReleased(KeyEvent event)
 		if (!config.isScreenshotEnabled())
 			return;
 
+		if (event.getKeyCode() == KeyEvent.VK_CONTROL)
+			this.CONTROL_KEY_DOWN = false;
+
 		if (event.getKeyCode() == KeyEvent.VK_INSERT)
 		{
 			plugin.takeScreenshot(TIME_FORMAT.format(new Date()));
```
Comment (Tobi1202) [inline runelite-client/src/main/java/net/runelite/client/plugins/screenshot/ScreenshotInput.java]: Is this on purpose? Insert is still bound to take a screenshot on release.

Comment (joshuapeters) [reply runelite-client/src/main/java/net/runelite/client/plugins/screenshot/ScreenshotInput.java]: @Tobi1202 keeping the insert keybind was purposeful, as I didn't want to remove functionality that many people might be used to.

However, pointing this out made me realize that the CONTROL_KEY_DOWN flag will not be set to false unless a user has the insert keybind option set to true, so I've fixed that!

**Comment by Adam-** [comment]
Instead of having everyone PR their favorite key for everything, I think we should allow people to set hotkeys for things.

**Comment by Gamer1120** [comment]
The hot key combination should be customizable in my opinion.

**Pull request closed without merging**